Clinical-Genomics/cg
# Update/set microbial samples apptag in LIMS via cli, resolves issue #296

This PR fixes the Changing microbial apptags in LIMS, resolves #296

How to test set apptag:
1. us
1. `cg set microbial-sample [LIMSID] [SIGNATURE] -a/--apptag [apptag]`

Expected outcome:
```
Application tag for sample [LIMSID] set to [apptag].
Comment added to sample [LIMSID]
update LIMS/application
```
microbial sample apptag has been set to the specified, an comment has been added to the order specifying who did what and when
Apptag has been set to [apptag] in LIMS for sample with [LIMSID]
